nss-day-cohort-48/rare-server-polite-pirates
# Mrw single post

#### Changes Made

1. Modified file `post/request.py` to include `get_single_post` and functionality. 
2. Modified file `post/__init__.py` to include `get_single_post` and functionality. 
3. Modified file `request_handler` to include `get_single_post` and functionality. 
4. Modified Directory `models/Categoies` to `models/categories`
   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout mrw-single-post
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   > run debugger in code
   >run postman http://localhost:8088/posts/3   whatever id you have.

- [x] You should see just that post with the matching ID
